Nos/Views/Settings/SettingsView.swift

```diff
@@ -217,6 +219,28 @@ struct SettingsView: View {
                     .padding(.vertical, 15)
             }
             .listRowGradientBackground()
+            
+            ActionButton(
+                title: .localizable.deleteMyAccount,
+                font: .clarityBold(.title3),
+                padding: EdgeInsets(top: 16, leading: 0, bottom: 16, trailing: 0),
+                depthEffectColor: .actionSecondaryDepthEffect,
+                backgroundGradient: .verticalAccentSecondary,
+                shouldFillHorizontalSpace: true
+            ) {
+                alert = AlertState(
+                    title: { TextState(String(localized: .localizable.deleteAccount)) },
+                    actions: {
+                        ButtonState(role: .destructive, action: .send(.deleteAccount)) {
+                            TextState(String(localized: .localizable.delete))
+                        }
+                    },
+                    message: { TextState(String(localized: .localizable.deleteAccountDescription)) }
+                )
+            }
+            .clipShape(Capsule())
```

Comment:
I think that this could be added into `ActionButton` directly instead of having its more manual corner rounding method, but I wasn't sure.

Reply:
Yes, that's what I meant. I can ask Sebastian about it.